src/components/List.jsx

```diff
@@ -20,17 +21,66 @@ export const List = ({ token }) => {
     return timeDiff < 24;
   };
 
+  const calcDaysSince = (transactionDate) => {
+    const timeNow = moment().format();
+    const date1 = moment(timeNow, 'YYYYMMDD HH:mm:ss');
+    const date2 = moment(transactionDate, 'YYYYMMDD HH:mm:ss');
+    const timeDiff = date1.diff(date2, 'days');
+
+    return timeDiff;
+  };
+
+  const cleanDate = (date) => {
+    const lastPurchasedDate = moment(date).format('MMMM Do, YYYY');
```

Comment:
something to consider when writing a function like this: we started off on a great foot by keeping the function name generic: "cleanDate", but the variables within the function indicate that this should be used for a specific purpose, "lastPurchasedDate" - this may confuse another dev should they come in here and want to work with this function, especially if it were to be longer than it is right now. A little change to something like "formattedDate" would keep this function generic 

```suggestion
    const formattedDate = moment(date).format('MMMM Do, YYYY');
```